wgpu-hal/src/gles/egl.rs

```diff
@@ -550,21 +550,25 @@ impl Inner {
         let supports_khr_context = display_extensions.contains("EGL_KHR_create_context");
 
         //TODO: make it so `Device` == EGL Context
```

Comment:
I think that todo comment is outdated. At least I have no idea what it wants

Reply:
It's there since the beginning, no idea either especially in the context of `context_attributes`